src/platform/src/test/java/org/geoserver/platform/resource/FileWrapperResourceTheoryTest.java

```diff
@@ -26,7 +30,13 @@
     protected Resource getResource(String path) throws Exception {
         File file = Paths.toFile(null, path);
         if (!file.isAbsolute()) {
-            file = Paths.toFile(folder.getRoot(), path);
+            if (folder.getRoot().getPath().startsWith("/") &&
+                    path.startsWith(folder.getRoot().getPath().substring(1))) {
+                //linux hack
```

Comment:
Well, the issue is that the whole FileWrapperResource is a hack. Resources are supposed to have a path relative to the store root (the data directory), not distinguishing between paths that start with "/" and paths that start without it. The FileWrapperResource is a resource without a store and it returns a regular file system path.
Because of this hack, this method distinguishes between relative and absolute paths (although according to the specs, there is no such thing with normal resources). However, in linux it cannot detect absolute paths if they do not start with a slash.